components/Market/utils.tsx

```
@@ -0,0 +1,202 @@
+import { MarketV2, Sale } from '../../types/universalis/MarketV2';
+
+export const enum Quality {
+  NormalQuality,
+  HighQuality,
+}
+
+export function shouldDisplayDiffTooltopEver(percentDiff: number): boolean {
+  return isFinite(percentDiff);
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)